scrapy/core/downloader/__init__.py

```diff
@@ -83,6 +86,7 @@ def __init__(self, crawler):
         self.middleware = DownloaderMiddlewareManager.from_crawler(crawler)
         self._slot_gc_loop = task.LoopingCall(self._slot_gc)
         self._slot_gc_loop.start(60)
+        self.per_slot_settings = self.settings.getdict('PER_SLOT_SETTINGS', {})
```

Comment:
I‘m thinking that we could just name the setting `DOWNLOAD_SLOTS`, in line with the `FEEDS` setting and the `download_slot` meta key. I‘m not against something like `DOWNLOAD_SLOT_SETTINGS`, and I think an argument could me made that adding `SETTINGS` makes it clear that there may be more slots that those defined there (since we are not defining slots, but slot settings), but `DOWNLOAD_SLOTS` is shorter and seems to be unambiguous enough (we probably should still clarify in the documentation that slots not in the setting may still be used).